phenopacketlab-frontend/src/app/component/pheno-creator/pheno-creator.component.html

```diff
@@ -1,7 +1,54 @@
 <div class="grid p-fluid m-5">
   <div class="field col-12">
-    <h1 class="header-step"><b>Phenopacket Creator for Rare Disease</b></h1>
-    <p-steps [model]="items" [readonly]="false" ></p-steps>
+    <h1 class="header-step">
+      <div class="flex justify-content-between flex-wrap m-5">
+        <div class="flex align-items-center justify-content-center">
+          <b>Phenopacket Creator</b>
+        </div>
+        <div class="flex align-items-center justify-content-center">
+          <p-toggleButton
+            [(ngModel)]="rareProfileSelected"
+            pTooltip="Toggle between Rare Disease steps and All available steps"
+            tooltipPosition="bottom"
+            [onLabel]="'Show all available steps'"
+            [offLabel]="'Show rare disease steps'"
+            styleClass="p-button-outlined"
+            (onChange)="updateProfile($event.checked)"
+          ></p-toggleButton>
+        </div>
+      </div>
+      
+    </h1>
+    
+    <p-steps
+      [model]="items"
+      [readonly]="false"
+      [(activeIndex)]="activeIndex"
+      (activeIndexChange)="onActiveIndexChange($event)"
+    ></p-steps>
   </div>
 </div>
 <router-outlet></router-outlet>
+<div class="flex justify-content-between flex-wrap m-5">
```

Comment:
Can you make these buttons sticky to the bottom?

Reply:
Just did it, let me know if you find that solution enough or if you have a more elegant solution for sticking these to the bottom of the step page